environment.yml

```diff
@@ -0,0 +1,24 @@
+name: bmorph
+channels:
+    - conda-forge
+dependencies:
+    - python>=3.5
+    - xarray
+    - pandas
+    - netcdf4
+    - numpy
+    - scipy
+    - dask
+    - toolz
+    - geopandas
+    - seaborn
+    - scikit-learn
+    - tqdm
+    - joblib
+    - networkx
+    - graphviz
+    - pygraphviz
+    - jupyter
+    - pip
+    - pip:
+        - tensorflow
```

Comment:
Do you need to specify tensorlfow>=2.0.0 here? Or just specify a version?

Reply:
Good thought - I'll update this.